### DISCUSSION TITLE
Map not loading on .Net MAUI 8 Android platform

### DISCUSSION THREAD

**Discussion by PopescuGeorge94** [Mapsui/Mapsui#2178] [General]
OpenStreetMap not loading tiles on .NET MAUI 8.

**Comment by pauldendulk** [comment]
Please write a title that describes the problem. Also mention which platform (iOS/Android/Windows).

Perhaps this is the user-agent problem:
https://github.com/Mapsui/Mapsui/issues/2172

Does it work properly with .NET 7 but the same sample not with .NET 8?

**Reply by PopescuGeorge94** [reply]
Sorry, on .net 7 MAUI Android platform everything works fine but on .Net 8 the screen becomes white.

**Reply by inforithmics** [reply]
I should upgrade SkiaSharp to 2.88.6 in your project, because 2.88.3 does not support .net 8 yet.

**Reply by PopescuGeorge94** [reply]
Hi, I installed the latest SkiaSharp package, but the error does not disappear.

**Reply by inforithmics** [reply]
Pull request to add .NET 8 Targets to Mapsui. When this is green I will investigate the specific android Problem if it still exists
https://github.com/Mapsui/Mapsui/pull/2179

**Reply by inforithmics** [reply]
I had a look at it in the .NET 8 pull request. I made following observations. Some samples work with OpenStreetMap and some don't.

I think this is fixed in this pull request. 
https://github.com/Mapsui/Mapsui/pull/2109
Because when a HttpRequest is handled in the UI Thread, Android throws an exception, and in this pull request
I put almost all Data Fetch Async Functions on the Background Thread.

**Reply by inforithmics** [reply]
Here are the Artifacts of this pull request.
https://github.com/Mapsui/Mapsui/suites/16479696300/artifacts/941782098
